webui/react/src/App.tsx

```diff
@@ -36,8 +36,13 @@ const AppView: React.FC = () => {
   useEffect(() => {
     setupAnalytics(info);
 
-    // Check to make sure the WebUI version matches the platform version.
-    if (info.version !== process.env.VERSION) {
+    /*
+     * Check to make sure the WebUI version matches the platform version.
+     * If the release is a release candidate or a dev tag, skip this notification.
+     * For example, check will run for `0.15.5` but not `0.15.5.dev3`.
+     */
+    const versionChunks = info.version.split('.') || [];
```

Comment:
suggestion I'd rely as least as possible on an untyped value and lean toward a direction where if the format changed we minimize the incorrect behavior aka fall back to showing the message for mismatched versions. considering this maybe checking the string for existence of `dev` would suffice? Kinda depends on whether we want to lean toward to showing this message or avoiding it.